docs/pages/update-spec/introduction.md

```diff
@@ -0,0 +1,184 @@
+{
+  id: string
+  createdAt: date
+  runtimeVersion: string
+  launchAsset: Asset,
+  assets: Asset[],
+  updateMetadata: {
+    updateGroup: string
+    branchName: string
+  }
+}
+```
+  * `id` The ID MUST uniquely specify the manifest, however the different headers may accompany identical IDs in a response.
+  * `createdAt` Time created is essential as the client selects the most recent update (subject to any contraints supplied in the `expo-manifest-filters` header).
+  * `runtimeVersion` Can be any string defined by the developer. It stipulates what native code setup is required to run the associated javascript bundle.
+  * `Asset` Provides information about the asset and where to obtain it.
+    * `hash` SHA256 hash of the file to guarantee integrity.
+    * `key` Key used by the bundler.
+    * `contentType` e.g. `application/javascript`, `image/jpeg`.
+    * `url` Location where the asset is hosted.
+  * `launchAsset` The asset that is used to launch the app.
+  * `assets` assets used by the app, such as photos or fonts.
+  * `updateMetadata` The metadata associated with an update can be used for filtering the update (branchName) and also for the creation of more helpful errors.
+
+### Error
+
+A conformant server MUST return a `400` error if a manifest is not found.
+
+## Server
+
+There are two functions for a conformant server:
+  * serve the correct [response](#response) to a given [request](#request)
+  * host the assets refenced in the [manifests](#manifest)
+These naturally can be hosted on different servers, but it is also possible to do it all on a single server.
+
+### Cacheing Policy
+
+Both the Manifest and the Asset endpoints MUST be served with a `cache-control` header set to an appropriately short period of time.
+
+For example:
+```
+cache-control: max-age=0, private
+```
+
+### Compression
+
+Assets SHOULD be capable being served with `zip` and `brotli` compression.
+The asset hosted at a particular URL MUST NOT be changed.
+
+### Channels and Branches
+
+A `branch` is a collection of `updates` stored on the server, ordered by creation time.
+
+The server MUST implement logic that maps a request to a `branch` determined by the request header `expo-channel-name` as well as any of optional server defined headers.
+
+The server MUST use the name of this branch to 
+  * define the `branchname` value of the `expo-manifest-filters` response header.
+  * define the `branchName` value of the `updateMetadata` JSON inside the [manifest](#manifest)
+
+The simplest case is to create a single branch of the same name for a given channel.
+
+More complex use cases would be to create multiple branches for a given channel. Then, using the request header `expo-rollout-token`, divide the users into groups that are mapped to different branches. This is useful for A/B testing or for gradually rolling out a new major version of an app.
+
+## Client
+
+See the [reference client](https://github.com/expo/expo/tree/master/packages/expo-updates)
```

Comment:
Above, when I read "client", I was imagining an app made with expo.

I think this should explain that an Expo project runs expo-updates, and together that makes up the "client"

Reply:
Maybe "client library"?